packages/neutrino-preset-node/index.js

```diff
@@ -35,11 +35,13 @@ module.exports = (neutrino) => {
   neutrino.use(compile, {
     include: [neutrino.options.source, neutrino.options.tests],
     babel: {
-      plugins: [require.resolve('babel-plugin-dynamic-import-node')],
+      plugins: [[require.resolve('fast-async'), { spec: true }], require.resolve('babel-plugin-dynamic-import-node')],
```

Comment:
@eliperelman https://github.com/mozilla-neutrino/neutrino-dev/pull/202/commits/354764382cd9681a31f0ad4496b4d1d0b654c208 how's this?